vpc-sample/tests.cloudbuild.yaml

```diff
@@ -22,9 +22,8 @@ steps:
   entrypoint: /bin/bash
   args:
   - '-c'
-  - | # Cloud Run Service name must be less than 63 characters
-    cat /dev/urandom | LC_CTYPE=C tr -dc 'a-z0-9' | head -c 15 > _short_id # Generate a random 15 character alphanumeric string (lowercase only)
-    gcloud alpha run deploy ${_SERVICE}-$(cat _short_id) \
+  - |
+    gcloud alpha run deploy ${_SERVICE}-$BUILD_ID \
```

Comment:
Likely out of scope, but maybe we don't need alpha here anymore.

Reply:
updated